ddtrace/internal/ci_visibility/recorder.py

```diff
@@ -43,6 +44,11 @@
 TEST_SKIPPING_LEVEL = "suite"
 
 
+if sys.version_info[:2] < (3, 6):
```

Comment:
You could simplify this bit of duplicated logic by putting it into the `ddtrace.internal.compat` module.